app/models/concerns/category_hashtag.rb

```diff
@@ -31,9 +31,8 @@ def query_from_hashtag_slug(category_slug)
     # depth supported).
     #
     # @param {Array} category_slugs - Slug strings to look up, can also be in the parent:child format
-    # @param {Array} cached_categories - An array of Hashes representing categories, Site.categories
-    #                                    should be used here since it is scoped to the Guardian.
-    def query_from_cached_categories(category_slugs, cached_categories)
+    # @param {Array} categories - An array of Category models scoped to the user's guardian permissions.
+    def query_from_cached_categories(category_slugs, categories)
```

Comment:
Does this method need to be change to reflect the fact the categories are no longer cached?

Reply:
Yeah probably not a great name...will think of something better